lib/mods/replace-methods.js

```diff
@@ -65,42 +65,42 @@ function getArgumentString(functionCall) {
 function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isCallExpression(node) &&
       isGlobal(node.getExpression())) {
-    let mod; let args; let type; let callee;
+    let moduleName; let args; let type; let callee; let propertyName;
 
     // property access methods
     if (nodeTypes.isPropertyAccessExpression(node.getExpression())) {
       const propertyAccess = node.getExpression();
-      name = propertyAccess.getSymbol().getEscapedName();
+      propertyName = propertyAccess.getName();
       type = 'method';
 
       let baseType = propertyAccess.getExpression().getType();
       const baseConstructSignature = baseType.getConstructSignatures()[0];
 
+      // the method is static if it has a constructSignature
       if (baseConstructSignature) {
         baseType = baseConstructSignature.getReturnType();
         type = 'static';
       }
 
-      mod = baseType.getSymbol().getEscapedName();
+      moduleName = baseType.getSymbol().getName();
       args = getArgumentString(propertyAccess.getParent());
       callee = propertyAccess.getExpression().getSymbol().getName();
 
-      if (type !== 'static' && isNamespaceMethod(propertyAccess, mod)) {
+      if (type !== 'static' && isNamespaceMethod(propertyAccess, moduleName)) {
         type = 'namespace';
-        callee = mod;
+        callee = moduleName;
       }
     } else { // window functions
       type = 'window';
-      mod = 'Window';
-      name = node.getExpression().getSymbol().getEscapedName();
-      callee = mod;
+      moduleName = 'Window';
+      propertyName = node.getExpression().getSymbol().getEscapedName();
```

Comment:
Should we be used EscapedName everywhere? If not, something should explain why we use it sometimes and not others.